src/sdk/namespace_fs.js

```diff
@@ -721,7 +721,7 @@ class NamespaceFS {
                         if (pos < results.length) {
                             results.splice(pos, 0, r);
                         } else {
-                            const stat = await native_fs_utils.stat_ignore_eacces(this.bucket_path, r, fs_context);
+                            const stat = await native_fs_utils.stat_ignore_eacces_enoent(this.bucket_path, r.key, fs_context);
```

Comment:
@shirady notice that if entry will not have a stat it might fail in _get_object_info. also better check that there are no issues with the limit. see https://github.com/noobaa/noobaa-core/pull/8751#discussion_r1940674457